### DISCUSSION TITLE
此处的全版本什么意思

### DISCUSSION THREAD

**Discussion by Chiloven945** [Meloong-Git/PCL#1789] [提问] [answered]
![image](https://user-images.githubusercontent.com/116699482/235990176-f5210d85-9f46-4472-9ea3-7276fd82a735.png)
感觉有点含糊的样子（？……

**Comment by iPodtouch6557** [comment]
1.0-snapshot1.20

**Reply by Chiloven945** [reply]
![image](https://user-images.githubusercontent.com/116699482/236595173-acf5555c-b7f7-4a42-a25e-e1f39bc93469.png)
……？

**Comment by WForst-Breeze** [comment, marked as answer]
此处 “全版本” 指该 Mod 支持 1.7 及以上所有主流 Minecraft 版本，这是刻意的设计
<details>
<summary>举例</summary>

![image](https://user-images.githubusercontent.com/110760354/236618190-c9c9494f-e21f-4f20-8cda-22460eb8f036.png)
![image](https://user-images.githubusercontent.com/110760354/236617935-1ccc2829-4fcb-4410-86f2-f3d6ccb22845.png)

---
![image](https://user-images.githubusercontent.com/110760354/236618159-41922a6c-72ae-4a6d-b9e4-894bfb7d597a.png)
![image](https://user-images.githubusercontent.com/110760354/236617947-9ac05180-0d56-4389-a870-54b76d04fae9.png)

---
![image](https://user-images.githubusercontent.com/110760354/236618121-e87e7357-fb2c-4763-8e43-3ae122468649.png)
![image](https://user-images.githubusercontent.com/110760354/236617962-cfba69aa-e83c-4f07-8763-62cebf010630.png)
</details>

**Reply by iPodtouch6557** [reply]
可是也有mod支持1.5.2forge啊，我认为这个全版本极易误解成 release 1.0~1.20 snapshot ，改一下比较好

**Reply by WForst-Breeze** [reply]
玩 1.7- 的玩家比较少，下载低版本 Mod 的玩家也很少，所以可能考虑到这点才把 1.7 及以上就归为全版本了。如果觉得有想法的话可以交个优化建议

**Reply by Chiloven945** [reply]
我之前也是这样想的，但是还是感觉不对，应该改为主要版本更加合适？
